src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/HealthCertificatesTabCoordinator.swift

```diff
@@ -204,6 +210,35 @@ final class HealthCertificatesTabCoordinator {
 		)
 		return topBottomContainerViewController
 	}
+	
+	private func exportCertificatesInfoScreen(
+		dismissAction: @escaping CompletionBool,
+		nextAction: @escaping CompletionVoid
+	) -> TopBottomContainerViewController<HealthCertificateExportCertificatesInfoViewController, FooterViewController> {
+		let healthCertificateExportCertificatesInfoViewController = HealthCertificateExportCertificatesInfoViewController(
+			viewModel: .init(
```

Comment:
I would personally prefer HealthCertificateExportCertificatesInfoViewModel() instead of .init

Reply:
as discussed, .init for long inner param types is okay